packages/react-components/react-theme-sass/package.json

```diff
@@ -38,6 +38,7 @@
   "dependencies": {},
   "exports": {
     ".": {
+      "style": "./sass/tokens.scss",
```

Comment:
Both [Vite](https://github.com/vitejs/vite/blob/a2af2f03b65b116989253dd11b16089a60cb6cbd/packages/vite/src/node/plugins/css.ts#L750-L752
) and [sass-loader](https://github.com/webpack-contrib/sass-loader/pull/647) in Wepback rely on `style` field. It's present in `package.json` itself, but it was missing in `exports`.